pkg/model/components/addonmanifests/awsloadbalancercontroller/iam.go

```diff
@@ -32,7 +33,17 @@ func (r *ServiceAccount) BuildAWSPolicy(b *iam.PolicyBuilder) (*iam.Policy, erro
 	clusterName := b.Cluster.ObjectMeta.Name
 	p := iam.NewPolicy(clusterName, b.Partition)
 
-	iam.AddAWSLoadbalancerControllerPermissions(p)
+	var enableUseOfWAFClassic bool
+	var enableUseOfWAF bool
+	if c := b.Cluster.Spec.AWSLoadBalancerController; c != nil {
+		if fi.BoolValue(c.EnableUseOfWAFClassic) {
+			enableUseOfWAFClassic = true
```

Comment:
This is probably easier written as `enableUseOfWAFClasic := fi.BoolValue(c.EnableUseOfWAFClassic)`

Reply:
Is that still the case even if these fields are now of type `bool` rather than `*bool`, per @hakman's earlier suggestion (https://github.com/kubernetes/kops/pull/13636#discussion_r870521607)?